tokio/src/fs/blocking.rs

```diff
@@ -74,7 +74,7 @@ where
                     }));
                 }
                 Busy(ref mut rx) => {
-                    let (res, mut buf, inner) = ready!(Pin::new(rx).poll(cx));
+                    let (res, mut buf, inner) = ready!(Pin::new(rx).poll(cx))?;
```

Comment:
Aligns the internals to factor in the new `Result<_, JoinError>` from using the task module.